rastervision_pytorch_learner/rastervision/pytorch_learner/semantic_segmentation_learner.py

```diff
@@ -209,12 +209,6 @@ def plot_batch(self,
         # (N, c, h, w) --> (N, h, w, c)
         x = x.permute(0, 2, 3, 1)
 
-        # apply transform, if given
-        if self.cfg.data.plot_options.transform is not None:
-            tf = A.from_dict(self.cfg.data.plot_options.transform)
-            x = tf(image=x.numpy())['image']
-            x = torch.from_numpy(x)
-
         # apply transform, if given
         if self.cfg.data.plot_options.transform is not None:
             tf = A.from_dict(self.cfg.data.plot_options.transform)
```

Comment:
I think ideally the transform should be applied on individual images in `x`, because albumentation transforms usually only work on single images rather than batches. This doesn't seem to matter in the case of normalization, but might cause problems if the user specifies some other transformation.

Something like:
```python
        if self.cfg.data.plot_options.transform is not None:
            tf = A.from_dict(self.cfg.data.plot_options.transform)
            x = x.numpy()
            imgs = [tf(image=img)['image'] for img in x]
            x = torch.from_numpy(np.stack(imgs))
```

Reply:
Yup, you're right.